### PR TITLE
add new check for multiple imports

### PR DESCRIPTION
**Pull request by JasonMWhite**
This PR introduces a new import check, restricting imports to one import per line:
```
from foo import bar  # OK
from foo import baz # OK
from foo import bar, baz  # forbidden
```
This PR also ran up against other rule checks, so I created a new assertion checker that allows you to limit which codes are verified. This makes tests less fragile (no interdependencies between codes). Also, consumers may disable individual codes, so it's important that we test a code in isolation, whether or not it also fails other checks.

@cfournie @honkfestival

### CODE REVIEW

**Comment by cfournie** [comment]
This isn't strictly mentioned in the Google Python Style Guide; should this instead be included in a separate Shopify-specific checker?

**Comment by JasonMWhite** [comment]
I think it is: https://google.github.io/styleguide/pyguide.html?showone=Imports_formatting#Imports_formatting

File: shopify_python/google_styleguide.py [hunk 1]
```diff
@@ -63,6 +63,9 @@ class GoogleStyleGuideChecker(checkers.BaseChecker):
                   'finally-too-long',
                   "The larger the 'finally' body size, the more likely that an exception will be raised during "
                   "resource cleanup activities."),
+        'C2610': ('%s imports multiple items in one import statement',
```
Comment (cfournie) [inline shopify_python/google_styleguide.py]: Specify `args` in the message if you want formatted output; currently it shows:

```
C:  8, 0: %s imports multiple items in one import statement (multiple-import-items)
```

Perhaps reword it to read:
```
Multiple items imported from %(module)s in one import statement.
```

And in `add_message` provide `args={'module': node.modname}`.

**Comment by cfournie** [comment]
Sorry, you're right, it is mentioned in the style guide.

File: shopify_python/google_styleguide.py [hunk 2]
```diff
@@ -63,6 +63,9 @@ class GoogleStyleGuideChecker(checkers.BaseChecker):
                   'finally-too-long',
                   "The larger the 'finally' body size, the more likely that an exception will be raised during "
                   "resource cleanup activities."),
+        'C6010': ('Multiple items imported from %(module)s in one import statement.',
```
Comment (cfournie) [inline shopify_python/google_styleguide.py]: These don't commonly end in a period and are usually shorter, e.g.:
```
shopify_python/google_styleguide.py:192:0 I0011(locally-disabled) Locally disabling no-member (E1101)
```

If printed this would look like:
```
shopify_python/google_styleguide.py:192:0 C6010(multiple-import-items) Multiple items imported from os in one import statement. (C6010)
```

Maybe rephrase as:

```
Statement imports multiple items from '%(module)s'
```

or


```
Multiple items imported from '%(module)s'
```

File: shopify_python/google_styleguide.py [hunk 3]
```diff
@@ -63,6 +63,9 @@ class GoogleStyleGuideChecker(checkers.BaseChecker):
...
+                  'multiple-import-items',
+                  'Separate imports into one item per line.')
```
Comment (cfournie) [inline shopify_python/google_styleguide.py]: You could expand upon this to explain the rationale.

File: shopify_python/google_styleguide.py (hunk 2 above)
Comment (honkfestival) [inline shopify_python/google_styleguide.py]: > `Statement imports multiple items from '%(module)s'`

👍

File: tests/shopify_python/test_google_styleguide.py [hunk 4]
```diff
@@ -11,6 +12,16 @@ class TestGoogleStyleGuideChecker(pylint.testutils.CheckerTestCase):
 
     CHECKER_CLASS = google_styleguide.GoogleStyleGuideChecker
 
+    @contextlib.contextmanager
+    def assert_adds_code_messages(self, codes, *messages):
```
Comment (honkfestival) [inline tests/shopify_python/test_google_styleguide.py]: 👍

**Comment by honkfestival** [comment]
LGTM pending @cfournie's requested changes
